### ISSUE TITLE
Team-Compass needs a content license

### ISSUE THREAD

**Issue by colliand** [2i2c-org/team-compass#782]
The content in 2i2c's doc sites should be shared under a license.

**Comment by choldgraf** [comment]
Any objections to a CC-BY license?

https://compass.2i2c.org/open-source/ip/#text-and-non-trademark-design-assets

If not I suggest we add a LICENSE file to the repository and also link it from the compass

**Comment by jmunroe** [comment]
CC-BY makes sense to me!

Do we do the same for https://github.com/2i2c-org/docs ?

**Comment by choldgraf** [comment]
Yes I think we should do this for all of our documentation etc. The way I think of it, the policy I've linked above is already our policy, and adding the license to our various docs etc locations is just implementing it. So I don't think we need to wait to make decisions here, we can just do it.
